Source/WTF/wtf/PlatformUse.h

```diff
@@ -376,6 +376,10 @@
 #define USE_AVIF 1
 #endif
 
+#if PLATFORM(IOS_SIMULATOR) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 160000
+#define USE_AVIF_FILE_TYPES_FOR_CG_BUG 1
+#endif
```

Comment:
Should not this macro be in PlatfromHave.h:

```
#define HAVE_CG_IMAGE_SOURCE_AVIF_IMAGE_TYPES_BUG 1
```

or something similar?

Reply:
Fixed.